packages/openapi-specification/openapi.yaml

```diff
@@ -1,742 +1,177 @@
-## this is a sample OpenAPI 3.0 specification!!!
 openapi: 3.1.0
 servers:
-  - url: 'http://petstore.swagger.io/v2'
+  - url: 'http://localhost:8080/api'
 info:
   description: >-
-    This is a sample server Petstore server. For this sample, you can use the api key
-    `special-key` to test the authorization filters.
+    This is the OpenAPI specification for the NSWI153 course project 'Webcrawler'.
   version: 1.0.0
-  title: OpenAPI Petstore
+  title: OpenAPI Webcrawler
   license:
     name: Apache-2.0
     url: 'https://www.apache.org/licenses/LICENSE-2.0.html'
```

Comment:
do we need licence?

Reply:
I forgot to remove it from the sample code, it is removed now.